src/tests/mocks/initializable_mock.cairo

```diff
@@ -7,7 +7,6 @@ mod InitializableMock {
     #[abi(embed_v0)]
     impl InitializableImpl =
         InitializableComponent::InitializableImpl<ContractState>;
-    impl InternalImpl = InitializableComponent::InternalImpl<ContractState>;
```

Comment:
to be honest i don't fully understand why `initialize()` is available for the initializable mock, since the internal impl is not instantiated nor embedded. Is it because the internal impl is being imported in the test file?

Reply:
Yes, it works because there is an implementation in context